webapp-django/crashstats/authentication/admin.py

```diff
@@ -3,12 +3,58 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
 from django.urls import reverse
 from django.utils.html import format_html
 
 from crashstats.authentication.models import PolicyException
 
 
+# Unregister the original UserAdmin and register our better one
+try:
+    admin.site.unregister(User)
+except TypeError:
+    pass
+
+
+@admin.register(User)
+class UserAdminBetter(UserAdmin):
+    """Improved UserAdmin.
+
+    This shows columns we care about and has links to PolicyException manipulation.
+
+    """
+
+    list_display = [
+        'email',
+        'first_name',
+        'last_name',
+        'is_active',
+        'is_hacker',
+        'is_superuser',
```

Comment:
We care about superusers. It's the set of people who maintain the site. Me and a few others. We don't care about staff, so I nixed that column.